src/main/scala/tapasco/task/ComposeTask.scala

```diff
@@ -41,9 +41,9 @@ class ComposeTask(composition: Composition,
                   features: Option[Seq[Feature]] = None,
                   logFile: Option[String] = None,
                   debugMode: Option[String] = None,
+                  private val effortLevel : String = "normal",
```

Comment:
Is there a reason for `effortLevel` being private and it using a default value instead of `Option[String]`?

Reply:
We cannot leave out the effort-level when writing the TCL script, so we need the default. With a given default, there's no need to use `Option` here.